src/main/scala/org/broadinstitute/dsde/firecloud/service/ServiceUtils.scala

```diff
@@ -0,0 +1,67 @@
+package org.broadinstitute.dsde.firecloud.service
+
+import akka.actor.ActorContext
+import akka.event.LoggingAdapter
+import spray.client.pipelining._
+import spray.http.HttpHeaders.{Cookie, Origin, RawHeader}
+import spray.http.StatusCodes._
+import spray.http.{HttpRequest, HttpResponse, RequestProcessingException, StatusCodes}
+import spray.routing.RequestContext
+
+import scala.concurrent.{ExecutionContext, Future}
+import scala.reflect._
+import scala.util.{Failure, Success}
+
+object ServiceUtils {
+
+  def listEntities(
```

Comment:
I like factoring this out into a common def. Though ...
1) This could be used for just about any get request; it doesn't have to be just for listing entities. Might want to rename it.
2) I could imagine passing in a return type for unmarshalling, and providing a default implementation of completeSuccessfully. Callers would only implement completeSuccessfully if they needed custom logic.
3) I could even imagine passing in the request method (i.e. get vs post) and an optional post payload.